src/backends/plonky2/primitives/ec/curve.rs

```diff
@@ -688,6 +744,13 @@ mod test {
         assert_eq!(p2, p3);
     }
 
+    #[test]
```

Comment:
Would you mind adding a test that does the roundtrip from point in the subgroup -> compress_from_subgroup -> decompress_into_subgroup -> assert_eq with the original point?
For some pseudorandom points?